docs/reference/snapshot-restore/restore-snapshot.asciidoc

```diff
@@ -1,187 +1,523 @@
 [[snapshots-restore-snapshot]]
 == Restore a snapshot
 
-////
+This guide shows you how to restore a snapshot. Snapshots are a convenient way
+to transfer data between clusters. You can also restore a snapshot to recover
+indices and data streams after deletion or a hardware failure.
```

Comment:
> Snapshots are a convenient way to transfer data between clusters.

Saying this first suggests that transferring data is their primary purpose but really they're more often used for recovery. How about something like this?

```suggestion
This guide shows you how to restore a snapshot. Snapshots are a convenient way
to store a copy of your data outside of a cluster. You can restore a snapshot
to recover indices and data streams after deletion or a hardware failure. You
can also use snapshots to transfer data between clusters.
```